cmds/exp/cpu/doc.go

```diff
@@ -7,6 +7,27 @@
 // Synopsis:
 //     cpu [OPTIONS]
 //
+// Advisory:
+//     cpu connects to a remote machine and serves a namespace to it.
+//     That namespace can be a restricted file system (recommended)
+//     or everything up to and including your /.
+//     We do cover a lot of the simpler adversarial attacks, via
+//     private name space mounts and the nonce on the 9p socket, and a
+//     quick review of this code by an expert suggest that on systems
+//     on which your adversary would only be those running at the same
+//     privilege level as you, you may be ok.
+//
+//     You should not use this command to connect to a host that might
+//     have an untrusted, prvileged adversary, i.e. someone who might
+//     replace the remote version with a corrupted version, or might
+//     use a different attack to grab the nonce and mount your file
+//     system, changing files before the client cpu exits.
+//.
+//     cpu is best used to connect to a linuxboot machine with only one
+//     user, i.e. a non-time-shared machine. We are developing a cpu we
```

Comment:
When we are developing...

Reply:
I moved the sentenced, the When is not needed.